po/sv.po

```diff
@@ -3,20 +3,22 @@
 # This file is distributed under the same license as the systemd package.
 # Josef Andersson <josef.andersson@fripost.org>, 2015.
 # Sebastian Rasmussen <sebras@gmail.com>, 2015.
+# Andreas Henriksson <andreas@fatal.se>, 2016.
+#
 msgid ""
 msgstr ""
 "Project-Id-Version: systemd master\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2015-11-22 16:37+0100\n"
-"PO-Revision-Date: 2015-03-14 11:09+0100\n"
-"Last-Translator: Sebastian Rasmussen <sebras@gmail.com>\n"
-"Language-Team: Swedish\n"
+"PO-Revision-Date: 2016-09-29 11:58+0200\n"
+"Last-Translator: Andreas Henriksson <andreas@fatal.se>\n"
+"Language-Team: Swedish <>\n"
```

Comment:
You're right. All meta-changes are done by gtranslator and I guess it wanted a validly formatted email-adress in this field. I manually dropped the <> in my updated version just posted because while being a valid email address you won't reach anyone at it as it goes into void.